### PR TITLE
Fix docs regarding `USB_SUSPEND_WAKEUP_DELAY`

### PR DESCRIPTION
**Pull request by ThreeFx**
This matches the value found in the [documentation](https://github.com/qmk/qmk_firmware/blob/74bec84740a2cc06e4409d28886fafb6afad1ef6/docs/config_options.md).

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* #17316

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by ThreeFx** [comment]
Things left to consider: Should we remove all keyboard configurations that already set it to 200? Or should we leave it up to the respective maintainers?

**Comment by drashna** [comment]
If anything, the documentation should be changed here, not the default.  

https://github.com/qmk/qmk_firmware/pull/12081

Specifically, when this was introduced, the default was set to 200ms by default and it caused a lot of issues.  So it was revert to 0ms (disabled). However, the document didn't get updated to reflect the change.

**Comment by ThreeFx** [comment]
Makes sense, in this case I've fixed the documentation to reflect that change, with a little note added that tweaking `USB_SUSPEND_WAKEUP_DELAY` might be a good idea. Unfortunately, I don't speak Japanese, so I only added it to the English version.

**Review by drashna — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**
